setup.py

```diff
@@ -33,7 +33,7 @@ def build_extensions(self):
 
 def main():
     cpu_count = os.cpu_count() or 8
-    version = "20210811"
```

Comment:
Should be the date of release like in the current master branch: https://github.com/CoinAlpha/hummingbot/blob/master/setup.py#L36